### PR TITLE
Glasgow | 25-ITP-Sep | Fares Bakhet | Sprint 3 | coursework/sprint-3-stretch

### PR DESCRIPTION
**Pull request by Fares-Bakhet**
## Learners, PR Template

Self checklist

- [x] I have titled my PR with Region | Cohort | FirstName LastName | Sprint | Assignment Title
- [x] My changes meet the requirements of the task
- [x] I have tested my changes
- [x] My changes follow the [style guide](https://curriculum.codeyourfuture.io/guides/reviewing/style-guide/)

## Changelist

-Build up functions and check them.
-Pass Jest tests.

## Questions

No questions.

### CODE REVIEW

File: Sprint-3/3-stretch/credit-card-validator.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+console.log(validateCreditCard(1111111111111111)); // false
+console.log(validateCreditCard(1234567890123456)); // true
+
+// Last digit must be even
+console.log(validateCreditCard(1234567890123451)); // false
+console.log(validateCreditCard(1234567890123452)); // true
+
+// Sum of all digits must be greater than 16
+console.log(validateCreditCard(1111111111111110)); // false
+console.log(validateCreditCard(1234567890123456)); // true
```
Comment (cjyuan) [inline Sprint-3/3-stretch/credit-card-validator.js]: Could also consider testing the boundary cases. e.g.,
- sum of digits is 16
- numbers with 15 digits
- numbers with 2 distinct digits
